### PR TITLE
Fix date computation

### PR DESCRIPTION
**Pull request by jbkt**

### CODE REVIEW

**Comment by guillaumebort** [comment]
Ok let's use an `Instant` for the start parameter then. So you can use an `Instant` in the example.

**Comment by jbkt** [comment]
Done

File: timeseries/src/main/scala/com/criteo/cuttle/timeseries/package.scala [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-    def date(args: Any*): LocalDateTime = macro safeLiteralDate
+    def date(args: Any*): Instant = macro safeLiteralDate
   }
 
-  def safeLiteralDate(c: Context)(args: c.Expr[Any]*): c.Expr[LocalDateTime] = {
+  def safeLiteralDate(c: blackbox.Context)(args: c.Expr[Any]*): c.Expr[Instant] = {
     import c.universe._
     c.prefix.tree match {
       case Apply(_, List(Apply(_, Literal(Constant(dateString: String)) :: Nil))) =>
         scala.util.Try(ZonedDateTime.parse(dateString)) match {
```
Comment (guillaumebort) [inline timeseries/src/main/scala/com/criteo/cuttle/timeseries/package.scala]: This has to be changed accordingly.

Comment (jbkt) [inline timeseries/src/main/scala/com/criteo/cuttle/timeseries/package.scala]: Done

**Pull request merged**